packages/management-api/src/management-service.ts

```diff
@@ -122,7 +121,7 @@ export class ManagementService extends EventEmitter {
          * against a non-running pod or a pod that cannot connect via jolokia
          * Emit an update but only if the status has in fact changed
          */
-        this.emitUpdate({ uid, fireUpdate: fingerprint !== this.fingerprint(mPod.getManagement()) })
+        this.emitUpdate({ uid, fireUpdate: mPod.getFingerprint() !== mPod.calcFingerprint() })
```

Comment:
if these are the only places where the fingerprint is used, we can put the fireUpdate logic into the managed pod class instead of repeating it here. we might be also able to make the fingerprint property and the getter/calc methods private, which further contributes to making the managed pod api simpler.